src/screens/SwapScreen/InfoBlock.tsx

```diff
@@ -3,41 +3,72 @@ import { useTheme } from "@emotion/react";
 import styled from "@emotion/styled";
 
 import SettingsIcon from "@src/assets/icons/gear.svg?react";
-import Text, { TEXT_TYPES } from "@src/components/Text";
+import LightningIcon from "@src/assets/icons/lightning.svg?react";
+import ArrowUpIcon from "@src/assets/icons/arrowUp.svg?react";
+import Text, { TEXT_TYPES, TEXT_TYPES_MAP } from "@src/components/Text";
 import { media } from "@src/themes/breakpoints";
 
 import { SlippageSettings } from "./SlippageSettings";
+import { useStores } from "@stores";
+import BN from "@src/utils/BN";
+import { DEFAULT_DECIMALS } from "@src/constants";
+import { SmartFlex } from "@components/SmartFlex";
 interface InfoBlockProps {
   slippage: number;
   updateSlippage: (percent: number) => void;
 }
 
 export const InfoBlock: React.FC<InfoBlockProps> = ({ slippage, updateSlippage }) => {
   const theme = useTheme();
+  const [showDetails, setShowDetails] = useState(false);
   const [isSlippageSettingOpen, setSlippageSettingOpen] = useState(false);
+  const { swapStore, oracleStore } = useStores();
+  const exchangeRate =
+    BN.formatUnits(oracleStore.getTokenIndexPrice(swapStore.sellToken.priceFeed), DEFAULT_DECIMALS).toNumber() /
+    BN.formatUnits(oracleStore.getTokenIndexPrice(swapStore.buyToken.priceFeed), DEFAULT_DECIMALS).toNumber();
+
   return (
     <Root>
-      <InfoLine>
-        <Text type={TEXT_TYPES.BUTTON}>Slippage tolerance</Text>
-        <LeftBlock>
-          <Text color={theme.colors.textPrimary} type={TEXT_TYPES.INFO}>
-            {slippage}%
-          </Text>
-          <Icon onClick={() => setSlippageSettingOpen(!isSlippageSettingOpen)} />
-        </LeftBlock>
-      </InfoLine>
-      <InfoLine>
-        <Text type={TEXT_TYPES.BUTTON}>Exchange fee</Text>
-        <Text color={theme.colors.textPrimary} type={TEXT_TYPES.INFO}>
-          0.00003 ETH (0,1$)
+      <InfoLine onClick={() => setShowDetails(!showDetails)}>
+        <Text type={TEXT_TYPES.SUPPORTING_TEXT_NEW}>
+          1 {swapStore.sellToken.symbol} = <SpanStyled>{exchangeRate.toFixed(6)}</SpanStyled>{" "}
```

Comment:
Думаю лучше будет использовать toSignificant
```suggestion
          1 {swapStore.sellToken.symbol} = <SpanStyled>{exchangeRate.toSignificant(6)}</SpanStyled>{" "}
```